### DISCUSSION TITLE
ModuleNotFoundError for CosseratPlugin

### DISCUSSION THREAD

**Discussion by LuyuHuang-AO** [sofa-framework/sofa#3735] [Build / Config / Environment] [answered]
Hi,

It seems like the Cosserat plugin is loaded, however, it still shows the module is not found:

[INFO]    [PluginManager] Loaded plugin: /SOFA/build/install/plugins/SofaPython3/lib/libSofaPython3.so
[INFO]    [PluginManager] Loaded plugin: /SOFA/build/install/plugins/CosseratPlugin/lib/libCosseratPlugin.so
[INFO]    [SofaPython3] Added '/SOFA/build/install/plugins/CosseratPlugin/lib/python3/site-packages' to sys.path
[INFO]    [PluginManager] 73 plugins have been loaded from /SOFA/build/install/lib/plugin_list.conf.default
[INFO]    [SofaPluginManager] Loading automatically plugin list in /SOFA/build/install/config/loadedPlugins.ini
[INFO]    [PluginManager] 0 plugins have been loaded from /SOFA/build/install/config/loadedPlugins.ini

[ERROR]   [SofaPython3::SceneLoader] Unable to completely load the scene from file 'cosseratExample1.py'.
Python exception:
  ModuleNotFoundError: No module named 'cosserat'

At:
  /SOFA/example/cosseratExample1.py(13): <module>
  <frozen importlib._bootstrap>(241): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(883): exec_module
  <string>(6): <module>

Any help would be appreciated. Thank you!

**Comment by LuyuHuang-AO** [comment, marked as answer]
I checked sys.path and found it was not taking the actual path to the cosserat module. Correcting the path with PYTHONPATH resolved the issue.

**Reply by DebbieDebbieli** [reply]
I meet the same problem and try to add the path "plugins\CosseratPlugin\lib\python3\site-packages\Sofa" to PYTHONPATH. But problem still exists. Any other set?
